templates/home/class_schedule.html

```diff
@@ -50,28 +50,28 @@ <h3 class="text-center">Class Schedule</h3>
                 <h5 class="card-title text-center">Add new schedule</h5>
                 <div class="row mt-2 justify-content-center align-items-center">
                     <div class="col-md-3 mb-3">
-                        <label for="sch-day">Day wise</label>
+                        <label for="sch-day">Day</label>
                         <select name="day" class="custom-select d-block w-100" id="sch-day">
                             <option value="">Select Day</option>
-                            {% for day, day_name in day %}
+                            {% for day, day_name in days %}
                             <option value="{{ day }}">{{ day_name }}</option>
                             {% endfor %}
                         </select>
                     </div>
                     <div class="col-md-3 mb-3">
-                        <label for="sch-section">Section wise</label>
+                        <label for="sch-section">Section</label>
                         <select name="section" class="custom-select d-block w-100" id="sch-section">
                             <option value="">Select Section</option>
-                            {% for section, section_name in section %}
-                            <option value="{{ section }}">{{ section_name }}</option>
+                            {% for section in sections %}
+                            <option value="{{ section }}">{{ section }}</option>
```

Comment:
Do the same thing here also, as mentioned above.

Also, here all sections should be visible and not just the sections which are being used in some schedule. For this, you can do something like this:

```
all_sections = Section.objects.order_by('section_id')
active_sections = all_sections.exclude(schedule=None)
```
and use `all_sections` at this place and `active_sections` above.